contrib/kafka/config/broker/kafka-broker.yaml

```diff
@@ -52,20 +67,21 @@ spec:
   - port: 9092
     name: kafka
     protocol: TCP
+
 ---
+
 apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: zookeeper
```

Comment:
Wait, we're running zookeeper in a Deployment? I'd expect that we'd want a StatefulSet, possibly with a persistent volume of some sort?

(Probably a separate issue.)

I'm also wondering why we ship a complete kafka cluster in our yaml...

Reply:
We have this YAML so that people like me, who know nothing about Kafka, can still use the `kafka` provisioner (but not in a production environment!).

I don't know why `Deployment` was chosen over `StatefulSet`.